### DISCUSSION TITLE
Version 4.0

### DISCUSSION THREAD

**Discussion by CodeAsherPlayz** [PythonMathCalculator/python-math-calculator#36] [Publishing]
@AdithyaR07 what should we do for Version 4? I am thinking of doing new input commands. Maybe we streamline the cmd variable because I don't want the list to be too long...

**Comment by AdithyaR07** [comment]
Which version are you talking about . Sorry I was not able to understand.  Sent from Mail for Windows 10 From: CodeAsherPlayzSent: 30 January 2021 19:23To: PythonMathCalculator/python-math-calculatorCc: AdithyaR07; MentionSubject: [PythonMathCalculator/python-math-calculator] Version 4.0 (#36) @AdithyaR07 what should we do for Version 4?—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Reply by CodeAsherPlayz** [reply]
I am planning of releasing a new version... So this is a completely new subject.

**Reply by CodeAsherPlayz** [reply]
@AdithyaR07 I feel that we should add in the options of different variations of pi and add a systematic process of choosing what the user wants to calculate. How do you feel about this?

**Comment by AdithyaR07** [comment]
Yeah , it sounds good. Let's go ahead with it.
